edalize/yosys.py

```diff
@@ -1,7 +1,12 @@
+
```

Comment:
Can you remove this empty line?

Reply:
why is this necessary?

Reply:
Thank you for your update to this pull request. You removed an empty line further down in the file, which is fine. However, the first line in this file is still an empty line, please remove it as well (it's unnecessary).

Reply:
what is the problem with it?